src/controllers/ignores/IgnorePhrase.hpp

```diff
@@ -44,6 +44,9 @@ class IgnorePhrase
 
     const std::unordered_map<EmoteName, EmotePtr> &getEmotes() const;
 
+    /**
+     * @brief containsEmote returns true if the ignored phrase replacement string contains a Twitch emotes
```

Comment:
I feel like there it should either be `contains Twitch emotes` or `contains a Twitch emote` because it seems like it is mixed singular and plural.